mobify/astro-scaffold
# [HYB-819] Integrate Preview Plugin

Add preview plugin into the scaffold for newly generated projects.

JIRA: **[HYB-819: Integrate preview plugin and make it a generator option](https://mobify.atlassian.net/browse/HYB-819)**
Linked PRs: **https://github.com/mobify/astro-scaffold/pull/123 | https://github.com/mobify/generator-astro/pull/41**

Requires https://github.com/mobify/astro-scaffold/pull/123 to be merged ☝️ first.

## Changes
- Include preview plugin with a toggle flag set to `false` by default (since Velo is not an adaptivejs/mobifyjs site)

## How to test-drive this PR
- Run the scaffold and ensure it still builds
- You may also run with the useAppPreview flag set to `true` in the `BaseConfig.js` file to see that the preview plugin does show up.

## TODOS:
